Bottom: 3326cb42c0239be2dde5eb880141dbb17b8e5904
Top:    0e5ce3fa891f8fc70adc486ccfd78c0d426e949a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 15:58:40 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
